README.md

```diff
@@ -1,5 +1,11 @@
 # telegram-bot
-Telegram bot server for RC Dining Bot
+Telegram bot server for RC Dining Bot to replace the previous dysfunctional bot which ceased to operate after
+change of data format in end of 2019.
+
+## Requirement
+* Set up PostgreSQL database locally/online
+* Configure .env file
```

Comment:
Probably should commit a `.env.example` to list out the required environment variables (and their default values, if any)

Reply:
as in drop some dummy data or place holder in the .env.example? Sounds like a good practice

Reply:
It serves as a template so that those who are setting up the environment for the first time can just `cp .env.example .env` and fill in the values there (instead of going through the entire code base and look for any environment variables)

Reply:
ok